src/EntityServer.ts

```diff
@@ -117,6 +124,15 @@ class EntityServer extends AssignmentClient {
         return this.#_maxOctreePPS;
     }
 
+    /*@devdoc
+     *  Triggered when an entity data packet is received.
+     *  @function EntityServer.addedEntity
+     *  @returns {Signal}
```

Comment:
Document as `@callback` without a `@returns`. See the MessageMixer for example.

Reply:
Fixed.